extensions/interactions/ItemSelectionInput/directives/ItemSelectionInputValidationService.js

```diff
@@ -131,6 +131,17 @@ oppia.factory('ItemSelectionInputValidationService', [
                         'please select only one answer choice.')
                     });
                   }
+                } else if (rule.type === 'IsProperSubsetOf') {
+                  handledAnswers[choiceIndex] = true;
+                  if (ruleInputs.length < 2) {
+                    warningsList.push({
+                      type: WARNING_TYPES.ERROR,
+                      message: (
+                        'In answer group ' + (answerIndex + 1) + ', ' +
+                        'rule ' + (ruleIndex + 1) + ', ' +
+                        'Is proper subset, needs at least 2 options.')
```

Comment:
... `the "proper subset" rule must include at least 2 options.`

Reply:
Done @seanlip , Ill get other software to correct grammar and spellings ASAP. Ill make sure these typos don't occur often.